Monocle_User_Guide.md

```diff
@@ -0,0 +1,77 @@
+#Monocle User Guide
+
+## Monocle Concepts
+### Traces
+Traces are the full view of a single end-to-end application KPI eg Chatbot application to provide a response to end user’s question. Traces consists of various metadata about the application run including status, start time, duration, input/outputs etc. It also includes a list of individual steps aka “spans with details about that step.
+It’s typically the workflow code components of an application that generate the traces for application runs. 
+### Spans
+Spans are the individual steps executed by the application to perform a GenAI related task” eg app retrieving vectors from DB, app querying LLM for inference etc. The span includes the type of operation, start time, duration and metadata relevant to that step eg Model name, parameters and model endpoint/server for an inference request.
+It’s typically the workflow code components of an application that generate the traces for application runs.
+
+## Setup Monocle
+- You can download Monocle library releases from Pypi
+``` 
+    > python3 -m pip install pipenv
+    > pip install monocle-observability
+```
+- You can locally build and install Monocle library from source
+```
+> pip install .
+> pip install -e ".[dev]"
+
+> python3 -m pip install pipenv
```

Comment:
for local build and install , the following are not explicitly required. You can delete them (line 22 and 23). 
> python3 -m pip install pipenv
> pipenv install build